extensions-c9s.yaml

```diff
@@ -18,7 +18,9 @@ extensions:
   # https://github.com/coreos/fedora-coreos-tracker/issues/1504
   ipsec:
     packages:
-      - libreswan
+      # pin to 4.6 for now for https://issues.redhat.com/browse/OCPBUGS-43498
```

Comment:
@jlebon I'm not sure that CS9 has these older versions, I'm also not sure that OKD will notice the sort of problems that this is meant to fix anyway it happens in decently large clusters which use ipsec. Do you think we should drop this change?